performance-starterkit-jmeter/ps_files/convertjmx_altjmx.ps1

```diff
@@ -0,0 +1,155 @@
+Write-Output "`n`n**** Conversion of your JMX to ALT compatible - True ****"
+Write-Output "Starting the conversion of input JMeter file to ALT Compatible JMX"
+# Read file and convert it into String #
+try{
+    $samplejson = Get-Content $jmxlocation -Raw -ErrorAction Stop
+}
+catch {
+    Write-Output "`nError reading the file: either the filepath is incorrect or there is no file at the specified path - $jmxlocation"
+    exit 1
+}
+
+# CSV File Path Update #
+$Allcsvdataset = $samplejson -split "<CSVDataSet guiclass"
+foreach ($csvdataset in $Allcsvdataset) {
+    if($csvdataset.contains("CSVDataSet")){
+        # Write-Output $csvdataset
+        $csvdatasetcontent = $csvdataset -split "</CSVDataSet>"
+        # Getting filename #
+        $filename = $csvdatasetcontent[0] -split "<stringProp name=`"filename`">"
+        $filename = $filename[1] -split "</stringProp>"
+        # Write-Output $filename[0]
+        #Check if file exists
+        if (-not (Test-Path $filename[0])) {
+            Write-Output "`nError: The csv file does not exist at the specified path. Please check the file path and try again."
+            exit 1
+        }
+        # Get Filename from Filepath 
+        if ($filename[0].Contains("\")){   
+            $filenamenew = $filename[0] -split "\\"
+            $filenamenew = $filenamenew[-1]
```

Comment:
Should we check if filename is absolute before replacing? Azure Load Testing might support relative paths to the CSV file in future, which can be paired with zip file with nested dir